src/mon/OSDMonitor.cc

```
@@ -1491,8 +1491,11 @@ void OSDMonitor::share_map_with_random_osd()
   }
 
   dout(10) << "committed, telling random " << s->inst << " all about it" << dendl;
+
+  //get features of the peer, fall back to my quorum_con_features.
```

add a space after `//`

---

and the wording of "fall back" is misleading, would be better to put

> use quorum_con_features, if it's an anonymous connection.